doc/start: KRBD feature flag support note (#51492)
ceph/ceph: doc/start/os-recommendations.rst

  *at least* 4.19-based "longterm maintenance" kernel series.  If you can
  use a newer "stable" or "longterm maintenance" kernel series, do it.

>   For KRBD, a kernel of *at least* version 5.4 is the minimum necessary for

Reviewer: K is assumed since we are in the "Ceph Kernel Client" section:
```suggestion
  For RBD, a kernel of *at least* version 5.4 is the minimum necessary for
```

Author: It is so revised.

>   reasonable RBD feature flag support.

Reviewer: I'd move this to the "Older kernel client versions may not support ..." paragraph below.

>   reasonable RBD feature flag support.

Reviewer: I'd move this to the "Older kernel client versions may not support ..." paragraph below and:
```suggestion
  reasonable RBD image features support.
```

Author: > I'd move this to the "Older kernel client versions may not support ..." paragraph below and:

Your lightest touch commands, @idryomov.

  for kernel version guidance.

  Older kernel client versions may not support your `CRUSH tunables`_ profile
  or other newer features of the Ceph cluster, requiring the storage cluster to
  be configured with those features disabled. For RBD, a kernel of *at least*

>   version 5.4 is the minimum necessary for reasonable support for RBD image

Reviewer: Just for my education: aren't "at least" and "the minimum" redundant here?

Author: @idryomov, Yeah, this is probably a little pleonastic. To my native-English-speaking, thinking-about-English-every-conscious-moment-for-four-decades ear, this doesn't sound absolutely grammatically wrong, but it is definitely a pleonasm. I'll remove the "at least". Good looking out.

  for kernel version guidance.

  Older kernel client versions may not support your `CRUSH tunables`_ profile
  or other newer features of the Ceph cluster, requiring the storage cluster to

>   be configured with those features disabled. For RBD, a kernel of version 5.4

Reviewer: I have checked and it's actually 5.3, not 5.4.  Anthony had 5.4 in mind probably because it's a major LTS kernel:
```suggestion
  be configured with those features disabled. For RBD, a kernel of version 5.3 or CentOS 8.2
```

Reviewer: CentOS 8 is based on 4.18 kernel but major RBD changes from 5.3 were backported there.
